spec/features/request_spec.rb

```diff
@@ -58,4 +58,25 @@ def fill_in_required_fields
       expect(page).to have_text(@request.topic_of_workshop)
     end
   end
+
+  describe "show page" do
+    context "as an organizer" do
+      before(:each) do
+        profile = FactoryGirl.create(:profile)
+        organizer = FactoryGirl.create(:user, role: :organizer, profile: profile)
+        login_as(organizer, scope: :user)
+      end
+
+      it "should allow me to change the status to :accepted" do
+        request = FactoryGirl.create(:request, status: :open)
+        visit(request_path(request))
+        status = I18n.t(@request.status, scope: 'activerecord.attributes.request.statuses')
+        expect(page).to have_text(status)
+        click_link I18n.t('requests.form.accept')
+        request = Request.find(request.id)
+        expect(request.status.to_sym).to eq(:accepted)
```

Comment:
Tom pointed out that since this is a feature test, database access is really wanted here.